src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogTcpClient.cs

```diff
@@ -87,7 +82,7 @@ private async Task ConnectAsync()
             }
         }
 
-        public async Task WriteAsync(IEnumerable<LogEvent> events)
+        public async Task WriteAsync(IReadOnlyCollection<LogEvent> events)
```

Comment:
This change is unnecessary.

Reply:
Yeah, I think you are right, it was extra. It was in the contributor PR, but I'm not sure why. I'll remove the unneeded changes.